releasenotes/notes/fix-profile-max-event-float-93288688036f669f.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    profiling: fix an issue where ``stack_event.StackExceptionSampleEvent`` used a float instead of an int for it's max events.
```

Comment:
```suggestion
    profiling: fix an issue where ``stack_event.StackExceptionSampleEvent`` used a float instead of an int for its max events.
```

Reply:
lol, can you tell @astuyve used my suggested text here? 😆